at-quannguyen/W4/channel.rb

```diff
@@ -0,0 +1,22 @@
+module Channel
+  def add_channel(list, *param)
+    unless list.has_key?(param[0])
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/f8dbe605eafc53803b1d102f4a00dad2a5e916b9/at-quannguyen/W4/channel.rb#L3"></span>Style/PreferredHashMethods: Use <code>Hash#key?</code> instead of <code>Hash#has_key?</code>. (https://github.com/rubocop-hq/ruby-style-guide#hash-key)</td>
    </tr>
  </tbody>
</table>